File: 백준/Bronze/29725. 체스 초보 브실이/체스 초보 브실이.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n').map(el => el.trim());
const chess = {
    'K' : 0, 
    'P' : 1, 
    'N' : 3, 
    'B' : 3, 
    'R' : 5, 
    'Q' : 9};
let count = 0;
for(let i = 0; i < 8; i++){
    for(let j = 0; j < 8; j++){
        let c = input[i][j];
        if(c === '.') continue;
        if(chess[c]) count += chess[c];
        else count -= chess[c.toUpperCase()];
    }
}
console.log(count);

File: 백준/Bronze/29725. 체스 초보 브실이/체스 초보 브실이.py
score = {'K' : 0, 'P' : 1, 'N' : 3, 'B' : 3, 'R' : 5, 'Q' : 9, 'k' : 0, 'p' : -1, 'n' : -3, 'b' : -3, 'r' : -5, 'q' : -9}
count = 0
for i in range(8) : 
    borad = list(map(str, input()))
    for item in borad :
        if item == '.' : 
            continue
        count += score[item]
print(count)